lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -49,6 +49,44 @@ def self.deliver_queue(args, options = {})
     end
   end
 
+  def self.deliver_synchronous(args)
+    log_header = "MIQ(#{name}.deliver_synchronous)"
+    options       = {:action => "Automate Request", :userid => 'system'}
```

Comment:
I think `:userid` should be `User.current_user || 'system'`  Are we handling the fully user identify so we use the proper tenant/automate domains?